src/Og.php

```diff
@@ -43,6 +44,9 @@ class Og {
    *     config definitions. Values should comply with FieldStorageConfig::create()
    *   - field_config: Array with values to override the field config
    *     definitions. Values should comply with FieldConfig::create()
+   *
+   * @return \Drupal\Core\Field\FieldConfigInterface
```

Comment:
Does this need to be field config interface? Would field definition interface suffice?